ai_tor_tell_you_an_appointment/lib/screens/faceSetting.dart

```diff
@@ -14,8 +14,8 @@ class FaceSettings extends StatefulWidget {
 
 class _FaceSettingsState extends State<FaceSettings> {
   Future<void> onImageSelectAndCrop(int index) async {
-    final ImagePicker _picker = ImagePicker();
-    final XFile? image = await _picker.pickImage(source: ImageSource.gallery);
+    final ImagePicker picker = ImagePicker();
```

Comment:
Why change it to picker? Because I want picker to be privileged variable (In format we use _ to indicate private variables) but that's fine...